attributes/git_hooks_core.rb

```diff
@@ -0,0 +1,4 @@
+node.default['sprout']['git']['hooks'] = {
+  'repository' => 'https://github.com/pivotal-cf-experimental/git-hooks-core',
```

Comment:
This looks like a set of pivotal internal hooks. I recommend that we leave this attribute empty so that users can specify `node['sprout']['git']['hooks']['repository']` unless we want to provide a default implementation under pivotal-sprout.